docs/ReleaseHistory.md

```diff
@@ -16,7 +16,14 @@
 - BRK, NEW: `SecretMasker.MaskSecrets` and `ISecretMasker.DetectSecrets` now accept an `Action<Detection>` callback (with a default value of `null`) to receive detections that result from the operation. This is binary-breaking only for `SecretMasker` callers and source-breaking as well for `ISecretMasker` implementations.
 - BRK: `Detection` is now sealed and immutable. All property setters and the copy constructor are removed.
 - BRK: `Detection`  no longer overrides `Equals` and `GetHashCode` nor implements `IEquatable<Detection>`. `Detection.Equals` and `Detection.GetHashCode` will therefore now be based on reference equality.
+- BRK, PRF: `IdentifiableScan` has been removed and its functionality has been merged into to `SecretMasker` which now scans the patterns its given as fast as possible, which will be significantly faster for identifiable key patterns.
+- BRK: `WellKnownRegexPatterns.*Iterator` methods have been removed. Use the corresponding `WellKnownPatterns.*` properties instead.
+- BRK: `WellKnownRegexPatterns.*` properties are now of stronger type`IReadOnlyList<RegexPattern>` instead of `IEnumerable<RegexPattern>`.
+- BRK: `SecretMasker.SyncObject` is now a read-only property instead of a public mutable field.
+- PRF: Add high-performance scanning for additional patterns: `SEC101/031.NuGetApiKey`, `SEC101/050.NpmAuthorKey`, `SEC101/110.AzureDatabricksPat`, and `SEC101/565.SecretScanningSampleToken`.
 - PRF: Fewer intermediate allocations are performed by `SecretMasker.MaskSecrets`.
+- NEW: `SecretMasker` is now capable of finding identifiable keys without relying on delimiting characters`.
```

Comment:
>NEW [](http://example.com/codeflow?start=2&length=3)

a `NEW` entry should precede all `PRF`. Also there is a trailing backtick at the end of this release note.